CMSgov/onemac
# remove references to user profiles table

Story: https://qmacbis.atlassian.net/browse/OY2-16597
Endpoint: https://dmdjcds6v3lwk.cloudfront.net/

### Details

Remove legacy `user-profiles` table from Dynamo to avoid it getting stale and messing up our current data store.

### Changes

- Remove table definition from Serverless template
- Remove migration logic that used to touch that table
- Migrate user data into `one-seed.json`

### Implementation Notes

- Updated `loadTestUsers.py` to use `one-seed` for test users. This requires us to split the now-combined full name to get the first and last for Cognito. Oh well.
- Removed `loadExistingUsers.py` and its dependencies as they seem to already be unused prior to this change.
- Removed Sabrina from test user data. No "real" people should be in there any longer.

### Test Plan

1. Observe the build and Cypress tests succeeding despite the changes
2. Test logging in as one or more of our test users and verify that you still have the expected access
3. Check that the `user-profiles` table for this branch has been deleted in the AWS Console
4. Observe [a fresh environment built off of this branch](https://github.com/CMSgov/onemac/actions/runs/1888701362) succeeding in the above as well
